pages/common/readonly.md

```diff
@@ -0,0 +1,18 @@
+# readonly
+
+> In Linux, the readonly command is used to create or modify read-only variables within a shell script.
+> When a variable is marked as read-only using readonly, its value cannot be changed or modified by subsequent commands within the script.
+> This can be useful when you want to ensure that a variable retains a constant value throughout the execution of a script.
+> More information: <https://man7.org/linux/man-pages/man1/readonly.1p.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/readonly.1p>.
```
We currently use manned.org for manpage links. See #5143.

Reply:
pushed the changes @reinhart1010

Reply:
This is also done @reinhart1010